GETTING_STARTED.md

```diff
@@ -584,6 +584,25 @@ factory :post do
 end
 ```
 
+Attribute overrides can be used to link associated child objects to their parent:
+```ruby
+# Define factories with associations
+FactoryBot.define do
+  factory :book do
+    title { 'A Farewell to Arms' }
+   end
+   
+   factory :chapter do
+    name { 'Chapter 1' }
+    book { association :book }
```

Comment:
```suggestion
    book
```

Although I think I would prefer to lean on the existing `post` and `author` factories rather than introducing new ones for this one section.